index.html

```diff
@@ -224,21 +224,21 @@ <h3>Full Name</h3>
                     <div class="custom-card-wrapper col-12 col-md-6 col-lg-4">
                         <div class="custom-card">
                             <div class="avatar-wrapper">
-                                <img src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
-                                    alt="" class="avatar">
+                                <img src="../../../../Pictures/Csaba Borbely.jpg"
```

Comment:
Csabi, imaginea asta nu exista pe repo-ul meu, deci ori faci un folder nou in care pui imaginea si comiti modificarea, ori iei imaginea cu "Copy image address" de undeva de unde ai uploadat-o (facebook, insta etc.)